addon/components/o-s-s/progress-bar.ts

```diff
@@ -0,0 +1,43 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+type ProgressBarSkins = 'primary' | 'warning' | 'success';
+
+interface OSSProgressBarArgs {
+  value: number;
+  label?: string;
+  valueIsVisible?: boolean;
+  skin?: ProgressBarSkins;
+  small?: boolean;
+}
+
+export default class OSSProgressBar extends Component<OSSProgressBarArgs> {
+  constructor(owner: unknown, args: OSSProgressBarArgs) {
+    super(owner, args);
+
+    assert('[component][OSS::ProgressBar] You must pass a numbered value between 0 and 100', args.value <= 100);
```

Comment:
Doesn't this raise an error if existence & type checking aren't done ?
maybe not but if it does, add `typeof args.value === 'number' && args.value >=0 && args.value <= 100`
(might as well check against negative values while we're there)